src/App.tsx

```diff
@@ -15,36 +17,93 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
+  length = 'length',
+  alph = 'alph',
+  default = '',
+}
+
+export const App = () => {
+  const [isReversed, setIsReversed] = useState(false);
+  const [sortType, setSortType] = useState(SortType.default);
+
+  const reverseList = () => {
+    if (isReversed) {
+      setIsReversed(false);
+    } else {
+      setIsReversed(true);
+    }
+  };
+
+  const resetList = () => {
+    setIsReversed(false);
+    setSortType(SortType.default);
+  };
+
+  const prepearedListOfGoods = [...goodsFromServer];
```

Comment:
There is a typo in the variable name `prepearedListOfGoods`. It should be `preparedListOfGoods` to correctly reflect the intended meaning.